extensions/emmet/src/balance.ts

```diff
@@ -56,30 +73,3 @@ function getRangeToBalanceOut(document: vscode.TextDocument, selection: vscode.S
 	}
 	return selection;
 }
-
-function getRangeToBalanceIn(document: vscode.TextDocument, selection: vscode.Selection, rootNode: HtmlNode): vscode.Selection {
```

Comment:
This function `getRangeToBalanceIn` is needed.
Take the case when the user selects an html node and starts balancing inwards. Then the `beforeEmmetSelection` array will be empty and the command would fail. In this case, we should use `getRangeToBalanceIn`